docs/src/howto-guides/010_add_self_signed_trusted.md

```diff
@@ -0,0 +1,27 @@
+# How to add self-signed certificate to trusted
+
+To manually add a self-signed certificate to be trusted by Linux system:
```

Comment:
I think we should give the steps for the two supported platforms and say that other linuxes might well be different. For example CentOS uses very different directories: /etc/pki

Reply:
@lines 13-17 mention for which systems those are, is it enough?

Reply:
No. The whole set of instructions is specific to the OS - not just installing ca-certificates.